lineage/Analyze.py

```diff
@@ -21,10 +21,12 @@ def preAnalyze(X, num_states):
     --------
     tHMMobj {obj}:
     """
-
+    fpi = kwargs.get('fpi', None)
```

Comment:
Can just put this in the argument list with `fpi=None` and accomplish the same thing.